hardhat/config/addresses.ts

```diff
@@ -35,6 +35,7 @@ export const getGelatoAddress = (network: string): string => {
     case "gnosis":
       return GELATO_GNOSIS;
     case "mumbai":
+    case "mumbaiDev":
```

Comment:
I think this is wrong, `mumbaiDev` has its own gelato diamond:
`0x266E4AB6baD069aFc28d3C2CC129f6F8455b1dc2`
https://github.com/gelatodigital/diamond/blob/master/deployments/mumbaiDev/Gelato.json